src/client/java/teammates/client/scripts/sql/SeedDb.java

```diff
@@ -195,6 +203,14 @@ private void seedStudents(int courseNumber, String courseId) {
                 student.setLastUpdate(rand.nextInt(3) > 1 ? null : getRandomInstant());
                 student.setRegistrationKey(studentRegistrationKey);
 
+                if (rand.nextDouble() >= PERCENTAGE_STUDENTS_WITH_ACCOUNT || googleIdToAccountForStudentsMap.containsKey(studentGoogleId)) {
```

Comment:
This might not work as expected, since the same "Account Google ID %s" is used for students in each course. If we are to seed many courses, it is very probable that every "Account Google ID %s" will end up with an account. 

One possible way would be to use `if (i % n == 0)` instead, so that every `n`th student has an account. 

I believe that we should also set `studentGoogleId` to `null` above, and assign the proper string within this if statement, since in the data migration script we have :
```
if (oldStudent.getGoogleId() != null) {
    userGoogleIdToUserMap.put(oldStudent.getGoogleId(), newStudent);
}
```